docs/DeveloperGuide.adoc

```diff
@@ -297,11 +302,27 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 |Priority |As a ... |I want to ... |So that I can...
 |`* * *` |new user |see usage instructions |refer to instructions when I forget how to use the App
 
-|`* * *` |user |add a new student |
+|`* * *` |form teacher |take the attendance of my students |know who is present for my class
 
-|`* * *` |user |delete a student |remove entries that I no longer need
+|`* * *` |form teacher |have a schedule tracking my events |know what I need to attend/do in a day
 
-|`* * *` |user |find a student by name |locate details of students without having to go through the entire list
+|`* * *` |form teacher |maintain of a list of students who have completed my homework |know who has not submitted my homework
+
+|`* * *` |form teacher |take down notes for student's behavior |track the behaviour of my students
+
+|`* * *` |form teacher |see the scores of my class |know the academic progress of my class
+
+|`* * *` |form teacher |add a new student |
+
+|`* * *` |form teacher |delete a student |remove entries that I no longer need
+
+|`* * *` |form teacher |find a student by name |locate details of students without having to go through the entire list
+
+|`* *` |form teacher |specify if a student is late or absent for class |know why my student is absent
+
+|`* *` |form teacher |update the details of my students |make necessary changes to my student's particulars
+
+|`* *` |form teacher |see usage instructions |refer to instructions when I forget how to use the App
```

Comment:
I like how you guys have included all the user stories, but personally i feel that the 1-star "|`*` |user with many students in the address book |sort students by name |locate a student easily" is a nice-to-have though.